### ISSUE TITLE
Add support for more optimization passes from SPIRV-Tools

### ISSUE THREAD

**Issue by haasn** [google/shaderc#373]
I noticed that SPIRV-Tools mentions quite a number of them: https://github.com/KhronosGroup/SPIRV-Tools#optimizer

Despite this, it seems only “Strip debug info” and “Unify constants” are exposed via shaderc. Would it make sense to expose the rest?

**Comment by antiagainst** [comment]
Yes, that's planned. Thanks for reporting! :)

**Comment by dneto0** [comment]
Agreed.   SPIRV-Tools is getting the ability to express higher-level sets of passes for various purposes. See https://github.com/KhronosGroup/SPIRV-Tools/pull/817 

Once that PR goes in, it would make sense to invoke those higher-level sets.
